src/types.ts

```diff
@@ -205,3 +207,12 @@ export enum UserIdleState {
   AWAY = "away",
   IDLE = "idle",
 }
+
+export type CanvasActionsState = Required<CanvasActions>;
+
+export type CanvasActions = {
+  saveScene?: boolean;
+  saveAsScene?: boolean;
+  clearCanvas?: boolean;
+  export?: boolean;
```

Comment:
```suggestion
  load?: boolean;
  save?: boolean;
  saveAs?: boolean;
  clear?: boolean;
  export?: boolean;
```
Lets keep the names short and also `load` will be part of this prop

Reply:
default value  all keys can be set as true

Reply:
If we keep the prop name short, we need a map of prop names to action names to disable it from the action level. That's fine, right?